client/ayon_core/tools/publisher/control.py

```diff
@@ -2472,12 +2672,20 @@ def _publish_iterator(self):
             if self._publish_up_validation and self.publish_has_validated:
                 yield MainThreadItem(self.stop_publish)
 
+            # TODO: Test the _is_interactive method in both
+            #  interactive and batch modes.
             # Stop if validation is over and validation errors happened
-            if (
-                self.publish_has_validated
-                and self.publish_has_validation_errors
-            ):
-                yield MainThreadItem(self.stop_publish)
+            if self.publish_has_validated:
+                if self._is_interactive:
+                    # In interactive mode, stop publish for
+                    # any validation errors
+                    if self.publish_has_validation_errors:
```

Comment:
Same here, we might want to use non blocking errors and _ignore_non_blocking_errors

Reply:
Same, ignore this ^^